pkg/wharfapi/client.go

```diff
@@ -26,3 +33,38 @@ type Client struct {
 // Deprecated: This type has been renamed to Client and may be removed in a
 // future release.
 type WharfClient Client
+
+func (c Client) Get(from, path string, q url.Values) ([]byte, error) {
```

Comment:
I see that you're reusing the parameter name `from` from the old `doRequest`. But I don't think it's a wise thing to replicate. Suggest chaning the parameter name throughout to something like `logMsg` or `endpointName`.

Or just remove it completely and only log the URL path. That would be sufficiently useful IMO.

Reply:
Going with only URL path 👍🏻